test/spec/openapi/route.js

```diff
@@ -419,3 +419,105 @@ test('cookie, query, path description', t => {
       })
   })
 })
+
+test('cookie and query with serialization type', t => {
```

Comment:
please use async/await in this test

Reply:
ready

Reply:
I think the `async / await` should be moved up one more level, like `async (t) => {}`